src/main/java/com/example/a1project/controllers/MainScreenController.java

```diff
@@ -1,16 +1,145 @@
 package com.example.a1project.controllers;
 
 import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
 
+import com.example.a1project.Task;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
 import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.geometry.Insets;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.GridPane;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
 
-public class MainScreenController extends SceneController {
+public class MainScreenController extends SceneController implements Initializable {
+	
+    @FXML
+    private GridPane currentTaskGridPane;
 
+    @FXML
+    private GridPane completedTaskGridPane;
+    
+    ObservableList<Task> currentTasks = FXCollections.observableArrayList();
+	ObservableList<Task> completedTasks = FXCollections.observableArrayList();
+    
 	public void addTask(ActionEvent event) throws IOException {
 		showNewTaskPopup(event);
 	}
 	
 	public void showAllTasks() {
```

Comment:
Good spotting, will be sure to add that in. We should also add comments for the signatures of each method as some of the method names are currently ambiguous or are very similar, e.g: addTask and addTaskToList

May need to open an issue to check whether all our existing methods also have JavaDoc signature comments.

Reply:
The addTask method was in fact redundant because it was never called by anything. It was also using outdated functionality that is currently implemented with addNewTask. So it's been removed